### DISCUSSION TITLE
what features require accessibility on android?

### DISCUSSION THREAD

**Discussion by avichou** [sannidhyaroy/Soduto#34] [Q&A] [answered]
trying to understand what accessiblity allows for on android 

(if you want to know why, it's because having ANY accessibility service breaks chrome "tap to search" feature , so i need to turn kde's accessiblity permissions off ) 

https://blog.google/products/chrome/4-ways-to-use-touch-to-search-on-chrome/

**Comment by sannidhyaroy** [comment, marked as answer]
Soduto doesn't have the features that needs Accessibility permissions on your Android. If you're using KDE Connect on your mac or Plasma desktop (or GSConnect on Gnome), you have features like `Receive Remote Keypresses`, `Mouse Receiver` that needs accessibility permissions to access your screen, interact with it and control it.
